### PR TITLE
Update copy: new user tutorial flow

### PR DESCRIPTION
**Pull request by Yihwan**

### CODE REVIEW

File: app/screens/TutorialScreens/MultipleChoiceScreen/constants.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const STEPS = [
     isTooltip: true,
   },
   {
-    text: "How many people do you think chose this B choice?",
+    text: "How many people do you think chose this answer?",
```
Comment (Yihwan) [inline app/screens/TutorialScreens/MultipleChoiceScreen/constants.ts:16]: there is a minor bug here that can be tracked separately: https://linear.app/gator/issue/PROD-134/multiple-choice-question-tutorial-seems-hard-coded-to-b-answer

**Review by marvinmarnold — APPROVED** [review]

File: app/screens/TutorialScreens/RevealScreen/RevealScreen.tsx [hunk 2]
```diff
@@ -128,13 +128,13 @@ const RevealScreen = ({
           <div className="fixed bottom-[0px] w-full p-6 bg-[#333333] flex flex-col gap-6 rounded-t-[32px] left-1/2 -translate-x-1/2 !max-w-[30rem] pointer-events-auto">
             <h3 className="text-base">Reveal answer?</h3>
             <p className="text-sm">
-              You would need to burn <b>5,000 BONK.</b>{" "}
+              This action burns <b>5,000 BONK.</b>{" "}
             </p>
             <div className="flex flex-col gap-2">
               <Tooltip
-                infoText="By spending a small amount of BONK, you can reveal the answer, and a chance to win the reward!
+                infoText="Revealing requires a small amount of BONK—but gives you the chance to win BONK rewards!
```
Comment (Yihwan) [inline app/screens/TutorialScreens/RevealScreen/RevealScreen.tsx:135]:
```suggestion
                infoText="Revealing burns a small amount of BONK—but gives you the chance to win BONK rewards!
```

File: app/screens/LoginScreens/NewUserScreen.tsx [hunk 3]
```diff
@@ -15,15 +15,14 @@ const NewUserScreen = () => {
 
           <div className="flex gap-[10px] items-center w-full justify-center">
             <p className="text-sm text-center">
-              We see that you&apos;re new here. Why not take a 30 second
-              tutorial to learn your way around the app?
+              New to Chomp? Why not take 30 seconds to learn how it all works?
```
Comment (Yihwan) [inline app/screens/LoginScreens/NewUserScreen.tsx:18]:
```suggestion
              New to Chomp? Why not take 30 seconds to learn how everything works?
```

File: app/screens/TutorialScreens/RevealScreen/RevealScreen.tsx [hunk 4]
```diff
@@ -165,7 +165,7 @@ const RevealScreen = ({
       <>
         <div className="px-4 w-full flex flex-col gap-4">
           <Tooltip
-            infoText="This is a TUTORIAL -  if this were a real question, and you answered Jupiter, you would have just won up to 10,000 BONK!  🎊"
+            infoText="If this were a real question (not a tutorial), you would have won up to 10,000 BONK for answering Jupiter! 🎊"
```
Comment (sno-mis) [inline app/screens/TutorialScreens/RevealScreen/RevealScreen.tsx:168]: 'Jupiter'

**Review by sno-mis — APPROVED** [review]

File: app/screens/TutorialScreens/RevealScreen/RevealScreen.tsx (hunk 4 above)
Comment (Yihwan) [inline app/screens/TutorialScreens/RevealScreen/RevealScreen.tsx:168]:
```suggestion
            infoText="If this were a real question (not a tutorial), you would have won up to 10,000 BONK for answering 'Jupiter'! 🎊"
```

**Pull request merged**